src/Model/ArticleVersion.php

```diff
@@ -283,6 +286,14 @@ public function getFunding()
         return $this->funding->wait();
     }
 
+    /**
+     * @return Sequence|Block[]
```

Comment:
https://github.com/elifesciences/api-raml/pull/216/files#diff-0351a4a0208c0ca05d9c16de3cc955f9R60 may constraint this to `Paragraph` rather than `Block`?

Reply:
I left these open to reduce code changes in the future if if the list of possibilities is ever expanded. Not adverse to changing though.